release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -63,6 +63,10 @@ You can incorporate example Azure Resource Manager (ARM) templates provided by R
 
 //See ../installing/install_azure_stack_hub/installing-azure-stack-hub-user-infra.adoc#installing-azure-stack-hub-user-infra[Installing a cluster on Azure Stack Hub using ARM templates] for details.
 
+[id="ocp-4-9-upgrade-pause-mhc"]
+==== Pausing machine health checks before updating the cluster
+
+During the upgrade process, nodes in the cluster might become temporarily unavailable. The machine health check might identify such nodes as unhealthy and reboot them. To avoid rebooting such nodes, {product-title} {product-version} introduces `cluster.x-k8s.io/paused=""` annotation to let you pause the `MachineHealthCheck` resources before updating the cluster. For more information, see xref:../machine_management/deploying-machine-health-checks.adoc#machine-health-checks-pausing[Pausing a MachineHealthCheck resource].
```

Comment:
Can it be all nodes (worker and master) in the cluster that might become unavailable? If it's only one type then maybe it should be spelt out.

Reply:
@slintes Kindly review the Release Notes content for adding the pause mechanism to MHC. Also, with reference to [@ogradyp's comment](https://github.com/openshift/openshift-docs/pull/36408#discussion_r711219103), I think that it is  applicable to all the nodes, but since we can't apply MHC to a machine with the master role, I might be mistaken. Kindly confirm.

Reply:
On one hand I'd say it depends on the MHCs the user creates, if it targets masters or workers or both.
On the other hand:
> since we can't apply MHC to a machine with the master role

I see this in the docs, but can't find this limitation in the code... 🤔 
@beekhof  do you know if MHCs can be applied to masters? (and if not, why / where the limitation is in code?)

Reply:
Ah, I think it is because master machines don't have a MachineSet as owner. So yes, at least today this affects workers only.

Reply:
@slintes I have update the content per our discussion. Kindly review and ack. Thank you